lib/cmd/fh2/account.js

```diff
@@ -183,27 +182,21 @@ function generateResource(dest, type, password, cb) {
     });
   };
   if (dest.toLowerCase() === "android") {
-    var list_url = "box/srv/1.1/dev/account/res/list";
-    common.doApiCall(fhreq.getFeedHenryUrl(), list_url, {"dest": "android"}, "Error listing account resources:", function (err, res) {
-      var dist_cert_found = false;
-      if (res.certificates && res.certificates.length > 0) {
-        for (var i = 0; i < res.certificates.length; i++) {
-          if (res.certificates[i].type === "distribution") {
-            dist_cert_found = true;
-            break;
-          }
-        }
-      }
-      if (dist_cert_found) {
+    var listUrl = "box/srv/1.1/dev/account/res/list";
+    common.doApiCall(fhreq.getFeedHenryUrl(), listUrl, {"dest": "android"}, "Error listing account resources:", function (err, res) {
+      var distCertFound = _.find(res.certificates, function (cert) {
+        return cert.type === "distribution";
+      });
+      if (distCertFound) {
         prompt("Warning: Found an existing distribution certificate for Android in your account. It will be overwritten and can not be recovered. Are you sure you want to continue ? \n (Y/N):", "", false, function (er, val) {
           if (val.toLowerCase() === "y" || val.toLowerCase() === "yes") {
-            genFunc("certificate", dest, password, cb);
```

Comment:
`genFunc` was actually using `cb` directly, since the name of its param was `callback`. Since it doesn't use more stuff from the closure it could be moved out of `generateResource` and get the param back.

I just went for the less-impacting change but I'm going to do that, since we don't have functions declared this way anywhere else.